prepareBundle.js

```diff
@@ -0,0 +1,12 @@
+const { execSync } = require('child_process');
+const package = require('./package.json');
+
+const version = package.version;
+const fileName = `openapi-cli.${version}.1111tar.gz`;
+const fileNameLatest = `openapi-cli.latest.tar.gz`;
+
+execSync(`tar -zcvf ${fileName} dist`);
+execSync(`tar -zcvf ${fileNameLatest} dist`);
+
+execSync(`aws s3 cp ${fileName} s3://openapi-cli-dist --profile redoc.ly`);
```

Comment:
remove `--profile`, profile names can be different on different devices.

Reply:
okay, but i'll add cli arg check for that, as there might be profiles instead of only 1 set of creds

Reply:
fixed